onyo/cli/tree.py

```diff
@@ -14,7 +14,9 @@
     'directory': dict(
         metavar='DIR',
         nargs='*',
-        help='Directories to list'
+        help=r"""
+            Directories to list
```

Comment:
Considering my earlier comment about trailing dots -- now there seem to be two help-texts that have one and two that do not. I think you should decide what is more typical and make it consistent eventually :)